git/foundations_git/git_basics.md

```diff
@@ -200,4 +200,5 @@ This section contains questions for you to check your understanding of this less
 
 This section contains helpful links to related content. It isn’t required, so consider it supplemental.
 
-- It looks like this lesson doesn't have any additional resources yet. Help us expand this section by contributing to our curriculum.
+- Learn more about Git through this [interactive web-based game](https://learngitbranching.js.org/) 
```

Comment:
We include this resource in a [later lesson](https://www.theodinproject.com/lessons/foundations-revisiting-rock-paper-scissors) we use to teach Git branches

Reply:
Got it, thanks!